src/Scientist/Internals/ExperimentInstance.cs

```diff
@@ -88,9 +90,14 @@ public ExperimentInstance(ExperimentSettings<T, TClean> settings)
 
             try
             {
-                // TODO: Make this Fire and forget so we don't have to wait for this
-                // to complete before we return a result
-                await Scientist.ResultPublisher.Publish(result);
+                Task task = Scientist.ResultPublisher.Publish(result);
+                PublishingTasks.TryAdd(task);
+
+                // Disable the warning, because the task is being tracked on the set, 
+                // and immediately removed upon completion.
+#pragma warning disable CS4014
+                task.ContinueWith(_ => PublishingTasks.TryRemove(task));
```

Comment:
I just thought a bit more about this and I think I'll need to address the exception handling a little better here.